ui/lib/replication/addon/templates/mode/secondaries/add.hbs

```diff
@@ -18,7 +18,7 @@
     </p>
   </div>
   <div class="field">
-    {{ttl-picker onChange=(action (mut ttl)) class="is-marginless"}}
+    {{ttl-picker onChange=(action (mut ttl)) initialValue="30m" class="is-marginless"}}
```

Comment:
this is a good interim fix, but it is a bit jarring to see `1800 seconds` as the default on the page (even though it does technically amount to 30 minutes), whereas everywhere else in the UI the TTL defaults to 30s. @chelshaw do you have any ideas about why this bug might be happening?

in any case, would be good to drop a comment reminder here + in JIRA to fix this bug & add it to our backlog 🐸

Reply:
Agreed - I'll go ahead and make a JIRA ticket for it and add it to the backlog. I'll also add a comment in the code.